static/app/components/events/featureFlags/utils.tsx

```diff
@@ -112,3 +112,24 @@ export const sortedFlags = ({
       return flags;
   }
 };
+
+export enum ProviderOptions {
+  LAUNCHDARKLY = 'LaunchDarkly',
+  OPENFEATURE = 'OpenFeature',
+}
+
+type Labels = {
+  pythonIntegration: string; // what's in the integrations array
+  pythonModule: string; // what's imported from sentry_sdk.integrations
+};
+
+export const PROVIDER_OPTION_TO_LABELS: Record<string, Labels> = {
```

Comment:
If the structure stays like this, you could make it type safe. Then nobody can forget to extend it when a new provider is added.
```suggestion
export const PROVIDER_OPTION_TO_LABELS: Record<ProviderOptions, Labels> = {
```